libimage/pull.go

```diff
@@ -379,6 +381,42 @@ func (r *Runtime) copyFromRegistry(ctx context.Context, ref types.ImageReference
 		return nil, err
 	}
 
+	// Avoid running out of time when running inside a systemd unit by
+	// regularly increasing the timeout.
+	if socketPath, _ := os.LookupEnv("NOTIFY_SOCKET"); socketPath != "" {
```

Comment:
Nit, why not use if socketPath,ok:=...'; ok {
}
That is why you use LookupEnv versus GetEnv?